views/addTransaction.handlebars

```diff
@@ -69,6 +69,11 @@
             <td>{{this.expenseName}}</td>
             <td>{{this.amount}}</td>
             <td>{{this.category}}</td>
+            <td>
```

Comment:
I see that you're using the query path parameter to set the method here. Are you using any middleware to convert the method like I did for the other ones?